TauAnalysis/MCEmbeddingTools/test/runtests.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -ex
+
+##____________________________________________________________________________||
+function die { echo $1: status $2 ; exit $2; }
+
+
+
+
+# echo '{
```

Comment:
since a bunch of files are created here , I suggest to go to the temp directory first, maybe even mkdir there for this test specifically so that someone else doesn't cleanup for you in a parallel running unit test
``` 
pushd ${LOCAL_TMP_DIR}
wd=embeddingTest
mkdir ${wd}
pushd ${wd}

everything else in this script

popd
rm ${wd}
popd
```